target_chains/ethereum/examples/oracle_swap/app/tsconfig.json

```diff
@@ -1,11 +1,12 @@
 {
+  "extends": "../../../../../tsconfig.base.json",
   "compilerOptions": {
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
-    "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "composite": false,
```

Comment:
I don't understand this option

Reply:
Aha, so this is another thing that took much of my time. In the ts base config I have the `composite: true` and that is one of the tricks to make ts compiler faster. However for the frontend here, it seems that it cannot build with that option and there. The issue is discussed [here](https://github.com/TypeStrong/ts-loader/issues/905).